software/control/src/AtlasFallDetector.cpp

```diff
@@ -73,7 +73,7 @@ void AtlasFallDetector::findFootIDS() {
 
 void AtlasFallDetector::handleAtlasStatus(const lcm::ReceiveBuffer* rbuf,
                        const std::string& chan,
-                       const atlas::status_t* msg) {
+                       const drc::behavior_t* msg) {
```

Comment:
behavior_t is a message containing only the behavior enum field in status_t
This behavior enum is the high level mode the BDI controller is in.

namely:

struct behavior_t
{
  int64_t utime;
  int32_t behavior;

  const int32_t BEHAVIOR_NONE=0;        //!< Robot currently not in any behavior; software freeze, controls disabled.
  const int32_t BEHAVIOR_FREEZE=1;      //!< All joints frozen in current position.
  const int32_t BEHAVIOR_STAND_PREP=2;  //!< Robot assumes a balanced posture, preparing for Stand.
  const int32_t BEHAVIOR_STAND=3;       //!< Statically stable stand.
  const int32_t BEHAVIOR_WALK=4;        //!< Dynamically stable walk.
  const int32_t BEHAVIOR_STEP=5;        //!< Statically stable, single-step walk.
  const int32_t BEHAVIOR_MANIPULATE=6;  //!< Statically stable stand, with upper body joints controlled by user.
  const int32_t BEHAVIOR_USER=7;        //!< Setpoints and gains set by user.
  const int32_t BEHAVIOR_CALIBRATE=8;   //!< Robot moves through various calibration motions, returns to Freeze.
  const int32_t BEHAVIOR_SOFT_STOP=9;   //!< "Safe" falling behavior, robot tucks its arms and legs and uses soft position gains
}